<launch>
  <node pkg="necst" name="check_node" type="ROS_check_node.py"/>
  <node pkg="necst" name="check_launch" type="ROS_check_launch.py"/>  
  <node pkg="necst" name="timer" type="ROS_timer.py"/>
  <node pkg="necst" name="antenna_move" type="ROS_antenna_move.py" output="screen" args="Actual"/>
  <node pkg='necst' name='authority_check' type='ROS_authority_check.py'/>
  <node pkg='necst' name='weather_status' type='S_ROS_weather.py'/>
  <node pkg='necst' name='pc_status' type='ROS_pc_status.py' args='necst'/>
  <node pkg='necst' name='tracking' type='ROS_tracking.py'/>
  <node pkg='necst' name='dome_tracking' type='ROS_dome_track_check.py'/>
  <node pkg="necst" name="worldcoordinate_onepoint" type="ROS_scan_onepoint.py"/>
  <node pkg="necst" name="worldcoordinate_otf" type="ROS_scan_otf.py"/>
  <node pkg="necst" name="worldcoordinate_planet" type="ROS_scan_planet.py"/>
  <node pkg="necst" name="worldcoordinate_linear" type="ROS_scan_linear.py"/>  
  <node pkg="necst" name="azel_list" type="ROS_azel_list.py"/>
  <node pkg='necst' name='check_alert' type='ROS_check_alert.py' output='screen'/>
  <node pkg='necst' name='alert_sun' type='ROS_alert_sun.py'/>
  <node pkg='necst' name='alert_weather' type='ROS_alert_weather.py'/>
  <node pkg='necst' name='alert_encoder' type='ROS_alert_encoder.py'/>
  <node pkg="necst" name="hosei_parameter" type="ROS_hosei_parameter.py"/>  
</launch>
